src/multiplayer-servers/multiplayer-services/steelshield/steelshield-for-agones.md

```diff
@@ -0,0 +1,68 @@
+# SteelShield for Agones
+
+SteelShield for Agones protects the Agones game server traffic from your GameFabric installation
+with powerful mitigations.
+
+## Pre-requisites
+
+In order to protect the Agones game server traffic, your GameFabric installation requires:
+
+- The SteelShield for Agones support enabled, usually indicated by the presence of the Protection/SteelShield navigation item.
+- Sites that support SteelShield, e.g. bare-metal.
+
+## Mitigations
+
+SteelShield provides mitigations that are tailored to check and let through only specific game or network traffic.
+To make use of the pre-defined mitigations you must create a new Protocol.
+
+## Protocols
+
+A Protocol – sometimes referred to as Protection Protocol, to distinguish it from a 
+network protocol such as UDP or TCP –
+describes your game traffic or what ever type of traffic you send through your game server ports.
+
+When you create a new Protocol, you must select a Mitigation and a network protocol.
+The network protocol that you select must be compatible with the Mitigation and their network protocol in order to be selectable.
+
+![create-protocol.png](images/create-protocol.png)
+
+::: warning
+You can restrict your Protocol to only support one UDP or TCP, even though the Mitigation would allow both.
+The other way round – allow both where only one is supported by the Mitigation – is not allowed.
+:::
+
+Once the Protocol is created, it is available to all Armada Sets, Armadas, Formations or Vessels.
+Nonetheless you need to actively select a Protection Protocol for each game server port of these entities.
+
+## Protect game server ports
+
+Any Armada Set, Armada, Formation or Vessel requires at least one game server port. 
+This port is usually to allow communication between your game server and your connecting game clients.
+It is common to even use multiple ports, one for the game engine, one for query services and so on.
+For each of them you can apply different Protocols.
+
+All ports are potentially vulnerable to attacks, so by setting a Protection Protocol,
+SteelShield applies a mitigation action to the traffic that goes through these ports.
+
+When the port is unprotected (`none`), SteelShield forwards any traffic without checking or filtering it.
+
+You can find the Protocol assignment right next to the port specification under 
+Vessel » Settings » Containers » Container Ports. It is the same for Armada Sets, Armadas and Formations.
```

Comment:
I left out Vessels because it is part of the example in the sentence before.

I generally find it a bit annoying to not have a common term for all of these four types. Hm.